### PR TITLE
Add AGPL licensing to source files

### PR DESCRIPTION
**Pull request by markettes**

### CODE REVIEW

**Comment by markettes** [comment]
Tool used: https://github.com/fbiville/headache

**Comment by RodriFS** [comment]
MVC/Razor files not included?

File: configuration-cs.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "headerFile": "./license-header.txt",
+  "style": "SlashStar",
+  "includes": ["src/**/*.cs"],
```
Comment (Jossec101) [inline configuration-cs.json:4]: We should include unit tests projects

Comment (markettes) [reply configuration-cs.json]: Done

**Review by Jossec101 — APPROVED** [review]

**Pull request merged**